packages/manager/apps/web/client/app/private-database/whitelist/private-database-whitelist.controller.js

```diff
@@ -192,5 +205,53 @@ angular.module('App').controller(
         );
       }
     }
+
+    getWebhostingNetworkStatus() {
+      this.whitelistService
+        .getWebhostingNetworkStatus(this.serviceName)
+        .then((res) => {
+          this.$scope.webhostingNetworkStatus = res;
```

Comment:
No other option that to store everything in `$scope`

Reply:
Fixed.